perception/lidar_centerpoint/config/centerpoint.param.yaml

```diff
@@ -13,14 +13,6 @@
     iou_nms_search_distance_2d: 10.0
     iou_nms_threshold: 0.1
     yaw_norm_thresholds: [0.3, 0.3, 0.3, 0.3, 0.0]
```

Comment:
Please move these post processing params in centerpoint to ml_packages.yaml as well as you did so in pointpainting (or let me know if there is any specific reason not to do so).

Reply:
Fixed at https://github.com/autowarefoundation/autoware.universe/pull/6591/commits/0109746b212f7989e1685365f5e6d54ae89d455e

Reply:
(Sorry, maybe my initial comment was wrong, I was confused somehow)
Our first motivation was to put the `score_threshold` in parameter file that is INDEPENDENT from the ml package.param.yaml, wasn't it?

That being said, I think those post processing parameters should be handled outside of ml_package.param.yaml

Reply:
No, our motivation is keep `score_threshold` parameter in ml_package.param.yaml for easier tuning purpose.

Reply:
Now it make sense. My understanding was wrong. Thanks for the clarification :pray: